ansible/hive-ocp-cluster-play/readme.md

```diff
@@ -63,25 +69,21 @@ Run the collection install command
 ## Create cluster
 
 When using variable files
+
 ```
-ansible-playbook -i inventory hive-ocp-cluster.yml
+ansible-playbook -i inventory hive-ocp-cluster-provision.yml
 ```
 
 When choosing to add variables to command line
+
 ```
-ansible-playbook -i inventory hive-ocp-cluster.yml -e "CLSUTER_NAME=your_cluster_name" -e "admin_task=provision" -e "cloud=aws"
+ansible-playbook -i inventory hive-ocp-cluster-provision.yml -e "CLUSTER_NAME=your_cluster_name" -e "cloud=aws"
 ```
 
 ## Destroy cluster
 
 Following pulls cluster name from common-vars.yml
 
 ```
-ansible-playbook -i inventory hive-ocp-cluster.yml -e "admin_task=delete"
-```
-
-Following uses command line to specify extra params that will overwrite what is in common-var.yml
-
-```
-ansible-playbook -i inventory hive-ocp-cluster.yml -e "CLSUTER_NAME=your_cluster_name" -e "admin_task=delete" -e "cloud=aws"
+ansible-playbook -i inventory hive-ocp-cluster-delete.yml
```

Comment:
Is there a reason that cluster name is no longer needed on delete?

Reply:
I updated the readme.  that delete is when you choose to use variable files instead of command line.